config/examples/rolling_updates.yml

```diff
@@ -14,22 +13,17 @@ base: &default
   policy_registry: true
   proxy_private_base_path: true
   service_mesh_integration: false
-  api_as_product: false
+  apicast_v2: true
```

Comment:
Could we sort these alphabetically?

Reply:
I have this in the PR description:

> P.S. I also had an idea to reorder the keys in the config files in the alphabetical order, but not sure how practical this is. To me it seems better to have them sorted, but if we decide to do so, I'd do it in another PR, otherwise it's impossible to compare the changes.

It's already not easy to track the changes to be honest :upside_down_face: 
But reordering would make it 100% impossible. So, yes, will do in another PR.